collections/_release_4_5/entry-general-editor-animation-player-quality-of-life-updates.md

```diff
@@ -0,0 +1,22 @@
+---
+type: entry
+section: general
+subsection: editor
+rank: 10
+importance: 2
+anchor: animation-player-quality-of-life-upgrades
+title: Animation player quality-of-life upgrades
+text: |
+  The animation player has received some needed love!
```

Comment:
```suggestion
  The animation player has received some much needed love!
```
The expression is always "much needed love", not "needed love", for some reason.